src/PointwiseFunctions/Hydro/StressEnergy.hpp

```diff
@@ -0,0 +1,135 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace hydro {
+
+/*!
+ * \brief The total mass-energy density measured by a normal observer, $E = n_a
+ * n_b T^{ab}$
+ *
+ * This quantity sources the gravitational field equations in the 3+1
+ * decomposition (see Eq. (2.138) in \cite BaumgarteShapiro).
+ *
+ * Perfect fluid contribution (Eq. (5.33) in \cite BaumgarteShapiro):
+ *
+ * \begin{equation}
+ *   E_\mathrm{fluid} = \rho_0 h W^2 - P
```

Comment:
I don't know what rho_0 is or what P is. We use `rho` for the baryon rest mass density and `D` for `rho W`, where `W` is the Lorentz factor. We use lower case `p` for the pressure.